copy-idls.sh

```diff
@@ -0,0 +1,6 @@
+# Only on first time
+# mkdir ./idls
+# cp ./target/idl/*.json ./idls
+
+anchor build
+cp ./target/idl/*.json ./idls
```

Comment:
Does anchor build accept outputting ids to another directory besides target?

Reply:
If not we have a scripts directory, your wrap the call to the script in package.json script, document in the contributing guid.

Reply:
You're right, `$ anchor build -i ./idls` does it